transparent-panels@germanfr/files/transparent-panels@germanfr/policies.js

```diff
@@ -142,6 +149,14 @@ MaximizedPolicy.prototype = {
 		this.lookup_windows_state(win.get_monitor());
 	},
 
+	_on_window_size_changed: function (wm, win, change) {
+		if (change === Meta.SizeChange.MAXIMIZE) {
```

Comment:
```suggestion
		if (change === Meta.SizeChange.MAXIMIZE || change === Meta.SizeChange.TILE) {
```

Reply:
I did merge this in but this does not solve all problems, yeah it will remove transparency when a window is tiled, but if the window is not tiled to fullscreen it still keeps the panel opaque.

It will need a little bit more logic to check that, but tbh I don't actually know what the expected behavior is.

Reply:
> but if the window is not tiled to fullscreen it still keeps the panel opaque.

How do you mean "not tiled to fullscreen"? If I tile a window to the left, right or corners the panels stay transparent. And if I drag a window from maximized state to a side, the panels get transparent as soon as the window gets resized.
Can you give me some steps to reproduce your case?

Thank you.

Reply:
You start fullscreen, then you tile the window to the left and around (topleft, top, topright etc) without exiting tile mode. That way the panel stays opaque.

The question is, should it?

Reply:
Ah, this works only with the keyboard shortcuts. I tried it always with the mouse to drag the window...

Yes, now I can reproduce the behavior. If I have a fullscreen window and press e.g. `Super+right` then the window gets tiled to the right side but the panel stays opaque.